apps/src/templates/projects/personal-projects-name-cell.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "color.scss";
+@import "font.scss";
+
+.infoButton {
+  background-color: transparent;
+  box-shadow: none;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  margin-left: 7px;
+  padding: 1px;
+}
+
+.infoButton:hover {
+  box-shadow: none;
+}
+
+.infoButton:focus {
+  border: solid;
+}
+
+.infoIcon {
+  color: $neutral_dark60 !important;
```

Comment:
Do we still need !important here?
Since we removed `!important` in a separate pr - should work fine now without !Important, try setting more specific css selector maybe? (like we do have in phase1-design-sysytem? (`#classroom-sections, #uitest-personal-projects  table td i `))

Reply:
I had the rule originally in `phase1-design-system` with `!important` because it was the only way to change the color of the icon, but then moved it back to be closer to the component. I'm good with moving it back to `phase1-design-system` if that is the preference - thanks!

Reply:
No no, I'm +1 to keep this styles here close to the component, I'm just asking if we need !important now or can we avoid using it by using more specific selector that will rewrite styles from `phase1-design-system.scss`

Reply:
Yes - it seems like we do need `!important` in this module since `phase1-design-system` seems to be applied last. So I can move it back to how it was if you like.

Reply:
Gotcha, lgtm then, +1 to merge as is right now!
If I'll be able to update it - will do it in a separate pr